### ISSUE TITLE
Import scss for each component inside tsx

### ISSUE THREAD

**Issue by danielwerg** [Aylur/astal#144]
**Have you read through the documentation?**

- [x] [Astal docs](https://aylur.github.io/astal/)
- [x] [Library references](https://aylur.github.io/astal/guide/libraries/references)
- [x] [FAQ](https://aylur.github.io/astal/guide/typescript/faq)

**Describe what have you tried so far**

Import shared styles (colors, vars) scss file from each component scss file and then import it in `src/scss/index.scss` that goes to astral. But you would need to write long import paths in `src/scss/index.scss` in file that you don't usually have open instead of `import ('./index.scss') in `index.tsx at each components dir.

**Describe your question**

Continuation from #143

```
my project
├── src
│   └── components
│       ├── Calendar
│       │   ├── index.tsx (imports index.scss, clearly doesn't work)
│       │   └── index.scss
│       ├── MediaPlayer
│       │   ├── index.tsx
│       │   └── index.scss
│       └── ...
└── scss
|   ├── index.scss
|   ├── styles.scss (imported in index.scss and each component)
|   ├── vars.scss (imported in styles)
|   └── colors.scss (imported in styles)
└── app.tsx
```

In react project you would import `index.scss` in `index.tsx` and bundler would handle everything for you. What's my options with ags and astlar?

**Comment by Aylur** [comment]
if you are using AGS, you can import your main scss file in app
```js
import style from "./scss/index.scss"

App.start({
  css: style,
})
```

if you are not using ags, you need to transpile scss first and feed the [stylesheet to app](https://aylur.github.io/astal/guide/typescript/theming#using-scss)

**Comment by danielwerg** [comment]
> main scss file

That's what my question is about, to have "main" scss file I need to have bunch of imports like this:

```scss
@import "../../path/to/component";
```

And that means I need to go to "scss" dir every time I add new component and add that line there instead of in neighboring `index.ts`. Once you project grows big enough it become harder to navigate. At that point I might as well just put _all_ scss files in `scss` dir next to `index.scss` to avoid problems with hot reload (https://github.com/Aylur/astal/issues/143).

---

After looking at about 10 repos of other people dotfiles, it's seems like most of the people put all scss under same directory (ex: https://github.com/Jas-SinghFSU/HyprPanel/blob/master/scss/main.scss) and one repo I found with file structure I'm trying to switch to https://github.com/matt1432/nixos-configs/blob/master/nixosModules/ags/config/style/main.scss.

I guess nobody has solution to the problem I explained in this issue.
